src/ReactiveUI.Tests/MessageBusTest.cs

```diff
@@ -117,18 +110,18 @@ public void RegisteringSecondMessageSourceShouldMergeBothSources()
         }
 
         [Fact]
-        public void MessageBusThreadingTest()
+        public async Task MessageBusThreadingTest()
         {
             var mb = new MessageBus();
             int? listenedThread = null;
             int? otherThread = null;
             int thisThread = Thread.CurrentThread.ManagedThreadId;
 
-            Task.Run(() => {
+            await Task.Run(() => {
```

Comment:
This wasn't that stable locally, but doing async/await rather than `Wait()` seemed to make it happier